froide/foirequest/forms/postal.py

```diff
@@ -142,3 +151,73 @@ def create_attachment(self, message, upload):
             partial(move_upload_to_attachment.delay, att.id, upload.id)
         )
         return att
+
+
+class ThinModelChoiceIterator(ModelChoiceIterator):
+    def __init__(self, field):
+        self.field = field
+        # basically empty the queryset so the thousands of choices are not rendered into context
+        self.queryset = field.queryset[:0]
+
+
+class ThinModelChoiceField(ModelChoiceField):
+    iterator = ThinModelChoiceIterator
+
+
+class PostalEditForm(MessageEditMixin, JSONMixin, forms.Form):
+    sent = forms.TypedChoiceField(
+        choices=(
+            (0, _("I have received the letter")),
+            (1, _("I have sent the letter")),
+        ),
+        coerce=lambda x: bool(int(x)),
```

Comment:
Is that safe?

Reply:
unsafe in what way? (I just copied it from elsewhere in the code.)